src/styles/variables.scss

```diff
@@ -0,0 +1,4 @@
+$calendar-day--size: 100px;
+$gap: 101;
```

Comment:
The `$gap` variable is set to `101`, which seems unusually high if it is intended to represent a spacing value. Double-check to ensure this is the intended value.